proto/v1/release_service.proto

```diff
@@ -122,6 +129,29 @@ message UndeleteReleaseRequest {
   ];
 }
 
+message CheckReleaseRequest {
+  // The database to dry-run the release.
+  // Format: instances/{instance}/databases/{database}
+  string database = 1 [
```

Comment:
Use repeated strings targets for a list of databases or a database group?

I also think the targets could be empty later for static analysis without hooking up actual database configuration and runtime.

Reply:
SG, done